cookbook/docs/ml_training.rst

```diff
@@ -34,7 +34,7 @@ ML Training
 
     .. link-button:: auto/case_studies/ml_training/spark_horovod/index
        :type: ref
-       :text: Forecast Sales Using Rossmann Store Sales Data
+       :text: Forecast Sales Using Rossmann Store Sales Data With Horovod and Spark
```

Comment:
If "w" in "with" is in lowercase in the README, it has to be in lowercase here as well.